dialects/postgres.yaml

```diff
@@ -26,18 +26,30 @@ dependencies:
   string: 
     https://github.com/substrait-io/substrait/blob/main/extensions/substrait/extensions/functions_string.yaml
 supported_types:
-  - i16
-  - i32
-  - i64
-  - fp32
-  - fp64
-  - bool
-  - str
-  - date
-  - time
-  - ts
-  - tstz
-  - interval
+  i16:
+    sql_type_name: smallint
+  i32:
+    sql_type_name: integer
+  i64:
+    sql_type_name: bigint
+  fp32:
+    sql_type_name: float4
+  fp64:
+    sql_type_name: float8
+  bool:
+    sql_type_name: boolean
+  str:
+    sql_type_name: text
+  date:
+    sql_type_name: date
```

Comment:
Why are these type names lowercase while most of the other dialects are capitalized?

Reply:
changed everything to lower case.